core/integration/client_tests.go

```diff
@@ -65,7 +65,9 @@ var (
 
 // TestEmptyGetAndUpdate verifies set/get semantics.
 func TestEmptyGetAndUpdate(ctx context.Context, env *Env, t *testing.T) {
-	signature.RegisterStandardKeyTypes()
+	if _, err := signature.RegisterStandardKeyTypes(); err != nil {
+		t.Errorf("RegisterStandaredKeyTypes(): %v", err)
```

Comment:
If RegisterStandardKeyTypes() fails, do you intend to continue with the rest of the test?  Just checking, as t.Errorf will log the error but then let the rest of the test continue.

Reply:
nope, it should be fatal